cmd/config/init/flags.go

```diff
@@ -98,3 +107,26 @@ func verifyTokenSupply(cmd *cobra.Command) error {
 
 	return nil
 }
+
+func verifyDecimals(cmd *cobra.Command) error {
+	decimals, err := cmd.Flags().GetUint(FlagNames.Decimals)
+	if err != nil {
+		return err
+	}
+	if decimals > 18 {
+		return fmt.Errorf("invalid decimals: %d. Must be less than or equal to 18", decimals)
+	}
+	return nil
+}
+
+func isValidDenom(s string) bool {
+	if len(s) != 3 {
```

Comment:
is it standard?

Reply:
No, but we add a 'u' in the beginning. What would make more sense for you?

Reply:
I meant why the SYMBOL is enforced to be 3 letters? can't it be SAMBA with uSAMBA?

Reply:
Ye, which restriction makes more sense to you: allowing only three to five alphabetic letters? I think this enforcement makes sense

Reply:
I think three to five is better